it/uk/gov/hmrc/excisemovementcontrolsystemapi/DraftExciseMovementControllerItSpec.scala

```diff
@@ -147,6 +147,12 @@ class DraftExciseMovementControllerItSpec extends PlaySpec
       postRequest(IE815).status mustBe FORBIDDEN
     }
 
+    "return forbidden (403) when the consignor is empty" in {
+      withAuthorizedTrader(consigneeId)
```

Comment:
This could be the consignorId as the consignor should be submitting (the consignee is covered in the other test)